aten/src/ATen/native/quantized/cpu/qlinear_prepack.cpp

```diff
@@ -300,6 +300,12 @@ static auto registry =
             c10::RegisterOperators::options().kernel<QLinearPackWeightInt8>(
                 DispatchKey::QuantizedCPUTensorId))
         .op("quantized::linear_prepack_fp16(Tensor W, Tensor? B=None) -> Tensor W_prepack",
+            c10::RegisterOperators::options().kernel<QLinearPackWeightFp16>(
+                DispatchKey::CPUTensorId))
+        .op("_quantized::linear_prepack(Tensor W, Tensor? B=None) -> Tensor W_prepack",
```

Comment:
why do you need to register these ops twice?

Reply:
It's a temporary solution and will be removed. Currently all lite interpreter ops are prefixed with "_" because the same ops may be already registered in JIT that were not searchable from lite interpreter. To make them build together for unit tests we have this naming hack cc @smessmer . The "_" will be removed after we have all ops searchable.